docs/02.API-REFERENCE.md

```diff
@@ -94,8 +94,8 @@ These option bits allow specializing Proxies with non-standard behaviour.
 These flags are recommended only for those trusted Proxies, whose handlers
 produce correct results.
 
-- JERRY_PROXY_SKIP_GET_CHECKS - skip [[Get]] result checks
-- JERRY_PROXY_SKIP_GET_OWN_PROPERTY_CHECKS - skip [[GetOwnProperty]] result checks
+- JERRY_PROXY_SKIP_RESULT_VALIDATION - skip result validation for [[IsExtensible]], [[PreventExtensions]], [[GetOwnProperty]],
+                                       [[DefineOwnProperty]], [[HasProperty]], [[Get]], [[Set]], [[Delete]] and [[OwnPropertyKeys]]
```

Comment:
What about the Get/SetPrototype other validations? Shouldn't we add those also?

Reply:
I have added the prototype handlers, but that should be enough.